Rename folders and files to photon-{something}

aminalhazwani opened the pull request:
@bwinton what do you think? I also have a couple of open questions regarding the `photon-colors/package.json`:

- if we change only the name/information architecture should we bump the version?
- should the `homepage` attribute point to the readme inside `/photon-colors` instead of the global one?

On a side note we could also create labels that we bind with the the `bugs/url` attribute.

Commit:
[fix] resolve #30

Commit:
[edit] photon-colors package.json to reflect the new name

bwinton commented:
Seems reasonable, but needs some fixing up. 😃


> if we change only the name/information architecture should we bump the version?

Yeah, probably. We need to for the sketch stuff anyways.

> should the homepage attribute point to the readme inside /photon-colors instead of the global one?

Yes. Good idea.

> On a side note we could also create labels that we bind with the the bugs/url attribute.

Please do, and update the `package.json`!

Thanks!

Commit:
Merge branch 'master' of github.com:FirefoxUX/design-tokens into issues/30

* 'master' of github.com:FirefoxUX/design-tokens:
  create sketch palette

# Conflicts:
#	colors/colors.gpl
#	photon-colors/photon-colors.css
#	photon-colors/photon-colors.js
#	photon-colors/photon-colors.less
#	photon-colors/photon-colors.scss

Commit:
v1.4.0 to include sketchpalette

aminalhazwani commented:
ok @bwinton, here some updates!

- pulled changes from `master`
- bumped the version to v1.4.0 `photon-colors/package.json`
- created a `tokens: colors` label
- changed the `bugs/url` attribute to `photon-colors/package.json`
- changed `homepage` attribute to `photon-colors/package.json`

bwinton commented on photon-colors/photon-colors.gpl:
```diff
@@ -0,0 +1,67 @@
+Name: Photon Colors
+# Photon Colors GPL Color Palette v1.4.0
+
+255 26 217 Magenta 50
```
We're picking up the GIMP colour renaming here, too? (I think that's okay, but wanted to make sure it was on purpose.)

aminalhazwani replied:
Yes, this is on purpose ☺️

bwinton commented on photon-colors/package.json:
```diff
@@ -20,7 +20,7 @@
+    "url": "https://github.com/FirefoxUX/design-tokens/issues?q=is%3Aissue+is%3Aopen+label%3A%22tokens%3A+colors%22"
   },
-  "homepage": "https://github.com/FirefoxUX/design-tokens#readme"
+  "homepage": "https://github.com/FirefoxUX/design-tokens/photon-colors#readme"
```
We should probably make these changes in the `photon-components/package.json` file, too.

aminalhazwani replied:
Good one! I also found out that the link needed to have the `/tree/master` path. I added to both `package.json` files and double-checked all the links.

bwinton reviewed (approved):
Mostly good, a couple of nits to fix, and then we can merge it! 🙂

Commit:
[fix] photon colors and photon components package.json links

bwinton commented on photon-components/package.json:
```diff
@@ -7,7 +7,7 @@
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/FirefoxUX/design-tokens.git"
+    "url": "git+https://github.com/firefoxux/design-tokens.git"
```
I think this is wrong. `FirefoxUX` should be in uppercase.

bwinton commented on photon-components/package.json:
```diff
@@ -19,7 +19,7 @@
   "license": "MPL-2.0",
   "bugs": {
-    "url": "https://github.com/FirefoxUX/design-tokens/issues"
+    "url": "https://github.com/firefoxux/design-tokens/labels/tokens%3A%20components"
```
No `?q=is%3Aissue+is%3Aopen+` on this one?

Commit:
[edit] links and fix link in bugs attribute photon-components

bwinton commented on photon-colors/build.js:
```diff
@@ -27,31 +27,30 @@ const formats = {
-# ${metadata.homepage}
+Name: Photon Colors
+# Photon Colors GPL Color Palette v${metadata.version}
 
```
Could we add the homepage back?

aminalhazwani replied:
Sure thing! I removed it because during my test it was never visible and it was the only file that featured it.

Commit:
[add] metadata homepage to GPL color palette

Commit:
[edit] bump version to v2.0.0
